src/coord/src/catalog/builtin.rs

```diff
@@ -1616,7 +1617,20 @@ WHERE c.relkind = ANY (ARRAY['r','p'])",
     needs_logs: false,
 };
 
-// Next id BuiltinView: 5036
+pub const PG_ACCESS_METHODS: BuiltinView = BuiltinView {
+    name: "pg_am",
+    schema: PG_CATALOG_SCHEMA,
+    sql: "CREATE VIEW pg_am AS \
+SELECT NULL::pg_catalog.oid AS oid, \
+    NULL::pg_catalog.text AS amname, \
+    NULL::pg_catalog.regproc AS amhandler, \
+    NULL::pg_catalog.char(1) AS amtype \
+WHERE false",
```

Comment:
tiny nit: backslash continuations unnecessary and not used in any of the other view defs